modules/mixcr/1.2/mixcr.smk

```diff
@@ -245,40 +247,36 @@ if CFG["igblastn"]:
 
     rule _update_mixcr_results:
         input:
-            db = rules._igblastn_run.output.db,
+            db = str(rules._igblastn_run.output.db),
             og_mixcr = CFG["dirs"]["mixcr"] + "{seq_type}/{sample_id}/mixcr.{sample_id}.clonotypes.{chain}.txt",
-            seq_info = rules._mixcr_to_fasta.output.seq_info,
+            seq_info = str(rules._mixcr_to_fasta.output.seq_info),
             script = CFG["igblast_scripts"]["igblastn2mixcr"]
         output:
-            txt = CFG["dirs"]["mixcr"] + "{seq_type}/{sample_id}/mixcr.{sample_id}.clonotypes.{chain}.status.txt"
+            txt = CFG["dirs"]["mixcr"] + "{seq_type}/{sample_id}/mixcr.{sample_id}.clonotypes.{chain}.igblast.txt"
         shell:
             "{input.script} -d {input.db} -m {input.og_mixcr} -o {output.txt} -s {input.seq_info}"
 
     rule _symlink_mixcr_update:
         input:
-            txt = rules._update_mixcr_results.output.txt
+            txt = str(rules._update_mixcr_results.output.txt)
         output:
-            txt = CFG["dirs"]["outputs"] + "txt/{seq_type}/mixcr.{sample_id}.clonotypes.{chain}.status.txt"
+            txt = CFG["dirs"]["outputs"] + "txt/{seq_type}/mixcr.{sample_id}.clonotypes.{chain}.igblast.txt"
+        wildcard_constraints:
+            chain = '[A-Z]+'
```

Comment:
A better wildcard constraint here is: 
```
chain = "|".join(RECEPTORS)
```
This makes a string of the possible values of the chain wildcards separated by the `|` symbol